kpops/components/base_components/models/to_section.py

```diff
@@ -79,20 +79,18 @@ def extra_topic_role(cls, values):
 class ToSection(BaseModel):
     """Holds multiple output topics
 
-    :param models: Data models
-    :type models: dict[str, Any]
     :param topics: Output topics
-    :type topics: dict[str, TopicConfig]
+    :param models: Data models
     """
 
+    topics: dict[TopicName, TopicConfig] = Field(
+        default={}, description=describe_attr("topics", __doc__)
+    )
     # TODO: really multiple models?
```

Comment:
Can we address this TODO somehow? I think it is very old.

Reply:
absolutely! will do in a follow up PR :)